smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/integration/TypeScriptIntegration.java

```diff
@@ -160,6 +160,25 @@ default void writeAdditionalFiles(
         // pass
     }
 
+    /**
+     * Write additional export statement to client root index.ts.
+     * This is necessary when {@link TypeScriptIntegration#writeAdditionalFiles} is also implemented.
```

Comment:
This isn't always necessary when using `writeAdditionalFiles`, as that method is used for writing out things that aren't exported, like the license file. Can you clarify this in the documentation and also have it state it's for exporting from the generated package? It would also be great if this had an example of its usage like the other methods.

Reply:
fixed in d1cc25d1e4d058808b2e9849da28859b1881de82